### ISSUE TITLE
BRs: Replace "CA" with "CA Operator" in section 8.1 

### ISSUE THREAD

**Issue by BenWilson-Mozilla** [cabforum/servercert#341]
Section 8 of the BRs says that the "CA" shall do XYZ.  Section 8.1 of the BRs says "the CA" shall obtain an audit report, etc., but shouldn't it say that the "CA operator" is required to do such things? I believe there is an issue here because the last paragraph of section 8.1 says,

If the CA does not have a currently valid Audit Report indicating compliance with one of
the audit schemes listed in Section 8.4, then, before issuing Publicly‐Trusted
Certificates, the CA SHALL successfully complete a point‐in‐time readiness assessment
performed in accordance with applicable standards under one of the audit schemes
listed in Section 8.4. The point‐in‐time readiness assessment SHALL be completed no
earlier than twelve (12) months prior to issuing Publicly‐Trusted Certificates and SHALL
be followed by a complete audit under such scheme within ninety (90) days of issuing
the first Publicly‐Trusted Certificate.

Most of this language was copied over from the EV Guidelines when the situation was that the EV certificate was being launched for the first time. Thus, a point-in-time readiness assessment was required. This no longer makes any sense. 

Shouldn't a "CA operator with a currently valid audit report ... " not be required to have a point-in-time readiness assessment?  

Shouldn't section 8.1 be edited to remove the "Point-in-Time readiness assessment" with something else? I.e., for the time it takes to get into root programs, doesn't it make sense to require a period-of-time audit instead of a point-in-time audit?

Are other fixes needed to this section and other parts of section 8?

**Comment by BenWilson-Mozilla** [comment]
@dzacharo - Remember when we went through the BRs and proposed [Ballot 188](https://cabforum.org/2017/03/02/ballot-188-clarify-use-term-ca-baseline-requirements/) to replace "CA" with "CA operator"?

**Comment by timfromdigicert** [comment]
Yes, we should get that one across the finish line.  Willing to help.  Again.

**Comment by BenWilson-Mozilla** [comment]
I've heard that some prefer the phrase "CA Owner".

**Comment by barrini** [comment]
This is one of the issues of using the CA term for everything. I don´t mind using CA owner or CA operator but this has to be aligned throught the whole document and maybe would be good to include it in the definitions section.

**Comment by ryancdickson** [comment]
+1 for a more consistent method of identifying the "flavors" of the term CA (i.e., "systems", "operators", and "owners").

Example: [Amazon ECDSA 384 M01](https://crt.sh/?q=dfe35c740cf41c0b053e2202ea5afc2f021f70bf90b26bc861fe1d9a0bfc4f1e)
- CA Owner: Amazon Trust Services
- CA Operator: DigiCert

**Comment by timfromdigicert** [comment]
There actually was a very advanced draft of this several years back.

The context of every use of CA was analyzed, and the vast majority were determined to refer to the "CA Organization".  A small minority of references really meant "CA Certificate".

Dimitris had the draft, I don't know if he still does.  I remember reviewing it but I doubt I have it any more.

**Comment by bcmorton** [comment]
This is starting to feel like a Forum topic or Forum definitions, since all working groups and baseline requirements should use the same language for CAs.

**Comment by RufusJWB** [comment]
> +1 for a more consistent method of identifying the "flavors" of the term CA (i.e., "systems", "operators", and "owners").
> 
> Example: [Amazon ECDSA 384 M01](https://crt.sh/?q=dfe35c740cf41c0b053e2202ea5afc2f021f70bf90b26bc861fe1d9a0bfc4f1e)
> 
> * CA Owner: Amazon Trust Services
> * CA Operator: DigiCert

Honestly, I have the feeling, we are complicating things here. In my view, this CA is nothing else than a cross-signed CA under the Amazon Root operated by a third party (Digicert) - like Siemens used to operate CAs cross-signed by Digicert/QuoVadis. It should be irrelevant, that this CA is dedicated to Amazon. Also, that Digicert operates their own Root-CAs should be irrelevant here. As a consequence, if there is an incident with this CA, I'd expect Amazon to open the bug at Bugzilla, as they have the overall responsibility for this CA hierarchy, and not DigiCert.

By following this approach, we don't need to introduce new terms and make things more complicated.

**Comment by dzacharo** [comment]
> There actually was a very advanced draft of this several years back.
> 
> The context of every use of CA was analyzed, and the vast majority were determined to refer to the "CA Organization". A small minority of references really meant "CA Certificate".
> 
> Dimitris had the draft, I don't know if he still does. I remember reviewing it but I doubt I have it any more.

I do have a version of the draft we worked at the time. The main goal was to clarify when we refer to "the CA" as "the organization that operates the CA" or as "the technical entity that signs certificates/CRLs/etc".

We put a lot of effort into this but in the end we decided to move to other priorities. It's definitely not on the top of the WG's list but if someone wants to volunteer to continue this work, I'd be happy to dig up my archive and find the latest draft.

**Comment by dzacharo** [comment]
> As a consequence, if there is an incident with this CA, I'd expect Amazon to open the bug at Bugzilla, as they have the overall responsibility for this CA hierarchy, and not DigiCert

I don't think this is a correct assumption. IMO an externally operated subordinate CA is responsible for handling its own incidents. The top CA is ultimately responsible, and would create its own incident, in case of lack of supervision or something other than the actual subordinate CA's incident.

**Comment by dzacharo** [comment]
@RufusJWB, can you please confirm that you participate as an Interested Party in the Server Certificate WG? Perhaps there was an oversight and your name/affiliation is not listed in https://cabforum.org/working-groups/scwg/#Members. The Forum welcomes any input but all contributions must be protected under the IPR agreement. Apologies if this has already been done.

**Comment by RufusJWB** [comment]
> @RufusJWB, can you please confirm that you participate as an Interested Party in the Server Certificate WG? Perhaps there was an oversight and your name/affiliation is not listed in https://cabforum.org/working-groups/scwg/#Members. The Forum welcomes any input but all contributions must be protected under the IPR agreement. Apologies if this has already been done.

I'm always mixing up the CA/B Forum Github Issues with the Mozilla Root Store Policy GitHub Issues. Im sorry

**Comment by dzacharo** [comment]
That's ok, no worries. Please ping me privately to review any possible challenges you face with signing the IPR agreement (either individually or as a company representative). I am gathering feedback from people that have difficulties signing the IPR agreement trying to get a better understanding of the issues.

Thanks.

**Comment by RufusJWB** [comment]
> That's ok, no worries. Please ping me privately to review any possible challenges you face with signing the IPR agreement (either individually or as a company representative). I am gathering feedback from people that have difficulties signing the IPR agreement trying to get a better understanding of the issues.
> 
> Thanks.

You've got mail from me to your j****@it.auth.gr address. I hope this is the correct one.

**Comment by barrini** [comment]
Move to Definitions WG
